lib/skooma/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Skooma
-  VERSION = "0.3.2"
+  VERSION = "0.3.3"
```

Comment:
Let's leave the current version 😄 
```suggestion
  VERSION = "0.3.2"
```